lib/logstash/gemfile.rb

```diff
@@ -0,0 +1,173 @@
+module LogStash
+
+  class GemfileError < StandardError; end
+
+  class Gemfile
+    attr_accessor :gemset
+
+    HEADER = \
```

Comment:
If I18n is available at the time this code is invoked, I would prefer it. Less for user-facing and more for having large constant strings all stuffed into a lookup table.